snapcraft.yaml

```diff
@@ -0,0 +1,24 @@
+name: sosreport
```

Comment:
For new packaging, I think using the project name of `sos` is a better approach, as the project is now more than the report functionality.

Reply:
:+1: